03/app.js

```diff
@@ -5,27 +5,57 @@ class Article extends React.Component {
     state = {
         comments: [],
     }
-    
+
+    changeHandler = e => {
+        const {tagName, value} = e.target;
+        this.setState({
+            [tagName.toLowerCase()]: value,
+        })
+    }
+
+    submitHandler = e => {
+        e.preventDefault();
+        const commentsCopy = [...this.state.comments];
+        console.log(this.state.comments)
+        this.addComment(commentsCopy);
```

Comment:
Wew. tej funkcji wykonujesz kopie itp. dlatego tutaj nie ma sensu tego wyrobić.
Wystarczy samo wywołanie tj. `this.addComment()`.